data/details.js

```diff
@@ -23,6 +23,15 @@ export const HERO_BANNER = {
   cfpUrl: 'https://www.papercall.io/pyconfhyd2025',
 };
 
+export const ABOUT_HYDPY = {
+  title: 'Meet our Organizers',
```

Comment:
`Meet the Organizers: HydPy` sounds more fitting to me. Can we use this instead?

Reply:
+1

Reply:
Updated title.